lib/accesslint/ci/cli.rb

```diff
@@ -0,0 +1,17 @@
+require "thor"
+
+module Accesslint
+  module Ci
+    class CLI < Thor
```

Comment:
What @geoffharcourt said. Also, there are ruby conventions around gem and class naming that would make the commands/files use `access_lint` instead of `accesslint`. The underscore bothers me somehow.